src/_P054_DMX512.ino

```diff
@@ -49,224 +55,212 @@
+              success = true;
+            }
 
-              else
-              {
-                int16_t index = param.indexOf('=');
-                if (index > 0)   //syntax: "<channel>=<value>"
-                {
-                  paramKey = param.substring(0, index);
-                  paramVal = param.substring(index+1);
-                  channel = paramKey.toInt();
-                }
-                else   //syntax: "<value>"
-                {
-                  paramVal = param;
-                }
+            else if (equals(param, F("off"))) {
+              memset(Plugin_054_DMXBuffer, 0, Plugin_054_DMXSize);
+              success = true;
+            }
 
-                value = paramVal.toInt();
-                PLUGIN_054_Limit (value, 0, 255);
+            else {
+              int16_t index = param.indexOf('=');
 
-                if (channel > 0 && channel <= Plugin_054_DMXSize)
-                  Plugin_054_DMXBuffer[channel-1] = value;
-                channel++;
+              if (index > 0) { // syntax: "<channel>=<value>"
+                paramKey = param.substring(0, index);
+                paramVal = param.substring(index + 1);
+                channel  = paramKey.toInt();
+              }
+              else { // syntax: "<value>"
+                paramVal = param;
               }
 
-              param = parseString(lowerString, paramIdx++);
+              value = paramVal.toInt();
+              PLUGIN_054_Limit(value, 0, 255);
+
+              if ((channel > 0) && (channel <= Plugin_054_DMXSize)) {
+                Plugin_054_DMXBuffer[channel - 1] = value;
+              }
+              channel++;
             }
-          }
-          else
-          {
-            //??? no params
-          }
 
-          success = true;
+            param = parseString(string, paramIdx++);
+          }
         }
 
-        break;
+        success = true;
       }
 
+      break;
+    }
+
     case PLUGIN_TEN_PER_SECOND:
+    {
+      if (Plugin_054_DMXBuffer)
       {
-        if (Plugin_054_DMXBuffer)
-        {
-          int16_t sendPin = 2;   //TX1 fix to GPIO2 (D4) == onboard LED
-
-          //empty serial from prev. transmit
-          Serial1.flush();
-
-          //send break
-          Serial1.end();
-          pinMode(sendPin, OUTPUT);
-          digitalWrite(sendPin, LOW);
-          delayMicroseconds(120);   //88µs ... inf
-          digitalWrite(sendPin, HIGH);
-          delayMicroseconds(12);   //8µs ... 1s
-
-          //send DMX data
-          Serial1.begin(250000, SERIAL_8N2);
-          Serial1.write(0);   //start uint8_t
-          Serial1.write(Plugin_054_DMXBuffer, Plugin_054_DMXSize);
-        }
-        break;
+        int16_t sendPin = CONFIG_PIN1; // ESP8266: TX1 fix to GPIO2 (D4) == onboard LED
+
+        // empty serial from prev. transmit
+        Serial1.flush();
+
+        // send break
+        Serial1.end();
+        pinMode(sendPin, OUTPUT);
+        digitalWrite(sendPin, LOW);

```

Comment:
I suspect this will not work, you really need to use direct GPIO access for this.
See https://github.com/letscontrolit/ESPEasy/tree/mega/lib/GPIO_DirectAccess
Also used this for other timing-critical stuff like 1-wire.

Reply:
Done